components/filters/enrolledstudents/plugin.class.php

```diff
@@ -91,10 +93,15 @@ public function print_filter(&$mform) {
                 $nameformat = get_string('fullnamedisplay');
             }
 
-            $sort = implode(',', order_in_string(get_all_user_name_fields(), $nameformat));
+            $namefields = [];
+            foreach (fields::get_name_fields() as $field) {
```

Comment:
what version is fields::get_name_fields land in core?

We probably need a new stable branch and not merge this into MOODLE_36_STABLE-catalyst

Reply:
Thanks @brendanheywood 

@mijiaa Can you please create a PR to `MOODLE_401_STABLE-catalyst` branch instead?

Reply:
Hi @brendanheywood it was 3.11. Thanks @tsmilan for creating the branch. I have change the destination branch for this PR.

Reply:
Thanks @mijiaa Is there a reason why you need an associative array of `$namefields`? 
Looks like `$sort = implode(',', order_in_string(fields::get_name_fields(), $nameformat));` will work just fine.

Reply:
Hi @tsmilan, thank you for pointing that out, I have amended the code as per your feedback.

Reply:
i have rebased as well